<?xml version="1.0"?>
<launch>
    <!-- カメラでキャプチャするためのパッケージを起動 --> 
    <!-- <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" > -->
        <!-- 起動するカメラのデバイス指定 -->
        <!-- <param name="video_device" value="/dev/video0"/> -->
    <!-- </node> -->

    <!-- 画像を表示するためのパッケージを起動 -->
    <!-- <node name="image_view" pkg="image_view" type="image_view" > -->
        <!-- image_viewのimageトピックがSubscribeするトピックの指定 -->
        <!-- <remap from="image" to="/usb_cam/image_raw"/> -->
        <!-- 画像の縦横比に合わせて自動で表示 -->
        <!-- <param name="autosize" value="true"/> -->
    <!-- </node> -->

    <!-- 作成したパッケージを起動 -->
    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="log">
        <!-- 起動するカメラのデバイス指定 -->
        <param name="video_device" value="/dev/video0"/>
    </node>
    <node name="yolov8_detect" pkg="image_recognition_study" type="yolov8_detect.py" output="screen"/>
</launch>